setup.py

```diff
@@ -54,6 +54,7 @@
             "descartes>=1.0.0",
             "geopandas>=0.8.0",
             "shapely>=1.6.0",
+            "pyarrow>=14.0.0"
```

Comment:
You should add a sentence in the PR detail about adding a new requirement, along with some context of why is it needed. Just for traceability.

Reply:
I'll add a sentence in the PR detail

Reply:
Added to PR description